### DISCUSSION TITLE
ACE Weather & Field Rations & Advanced Fatigue

### DISCUSSION THREAD

**Discussion by Rockfrog** [acemod/ACE3#8712] [Q&A]
Does the map temperature generated by ACE Weather impact the increased need for hydration and its impact on fatigue 'automatically', or do we need to manually adjust (increase) water consumption requirement and/or fatigue for desert maps?

**Comment by severgun** [comment]
Currently only player's movement affect stats.
https://github.com/acemod/ACE3/blob/e0df864a5d0778174491bd135947f474c93e9330/addons/field_rations/functions/fnc_update.sqf#L36-L44
You can create your own logic using
https://ace3mod.com/wiki/framework/field-rations-framework.html#33-adding-a-status-modifier
